openshift/Makefile

```diff
@@ -21,6 +21,7 @@ export GO_BUILD_LDFLAGS := -s -w -X '$(VERSION_PATH).gitCommit=$(GIT_COMMIT)'
 verify: ## Run downstream-specific verify
 	$(MAKE) tidy fmt generate -C $(DIR)/../
 	$(MAKE) manifests
+	$(MAKE) -C tests-extension verify-metadata
```

Comment:
Any reason we couldn't invoke `make -C openshift/test-extension verify` and add the `make verify-metadata` [there](https://github.com/openshift/operator-framework-operator-controller/blob/d9f86c18b08f563a30af18cf63bc7ce2305fa8c9/openshift/tests-extension/Makefile#L45) ?